### PR TITLE
Added element enabled check inside dropdown

### PR DESCRIPTION
**Pull request by SergeyRuzin**

### CODE REVIEW

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs [hunk 1]
```diff
@@ -77,6 +87,39 @@ public void CheckMultipleSelectedValues([NotNull] IMultiSelectDropdownWrapper wr
             wrapper.SelectedValuesTexts.Should().BeEquivalentTo(values.Rows.Select(x => x.Values.Single()));
         }
 
+        [Given("the \"(.*?)\" value (is|become) (enabled|disabled) in (.*?)")]
```
Comment (afomina-quantori) [inline src/Behavioral.Automation/Bindings/DropdownBinding.cs:90]: Could you please update the Bindings.md?

Comment (afomina-quantori) [inline src/Behavioral.Automation/Bindings/DropdownBinding.cs:90]: better to use (.+?)

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs [hunk 2]
```diff
@@ -77,6 +87,39 @@ public void CheckMultipleSelectedValues([NotNull] IMultiSelectDropdownWrapper wr
...
+        [Then("the \"(.*?)\" value should (be|become) (enabled|disabled) in (.*?)")]
+        public void CheckValueInDropdownIsEnabled([NotNull] string value, [NotNull] AssertionBehavior behavior,
+            bool enabled, IDropdownWrapper wrapper)
+        {
+            _attributeBinding.CheckElementIsDisabled(wrapper.Elements.Single(x => x.Text == value), behavior, enabled);
+        }
+
+        [Given("the following values (are|become) (enabled|disabled) in (.*?):")]
```
Comment (afomina-quantori) [inline src/Behavioral.Automation/Bindings/DropdownBinding.cs:98]: Please, ensure that this binding doesn't overlap with inside binding.

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs [hunk 3]
```diff
@@ -77,6 +87,39 @@ public void CheckMultipleSelectedValues([NotNull] IMultiSelectDropdownWrapper wr
...
+        [Then("the following values should (be|become) (enabled|disabled) in (.*?):")]
+        public void CheckMultipleValuesInDropdownAreEnabled([NotNull] string behavior, string enabled,
```
Comment (afomina-quantori) [inline src/Behavioral.Automation/Bindings/DropdownBinding.cs:100]: Can we use AssertionBehavior instance here instead of string?

**Review by afomina-quantori — CHANGES_REQUESTED** [review]

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs (hunk 3 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs:100]: I think probably not because this is used as a string in the called method below

**Review by PavelAAlexeevQ — APPROVED** [review]

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs (hunk 1 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs]: Updated

Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs]: Fixed

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs (hunk 2 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs]: It shouldn't because we're using "the" and it doesn't contain word "inside"

**Review by afomina-quantori — APPROVED** [review]

**Review by PavelAAlexeevQ — APPROVED** [review]

**Pull request merged**